codeGenerator/src/robotConfiguration/robotConfig.cs

```diff
@@ -78,6 +78,7 @@ robotVariants loadRobotConfiguration(string fullPathName)
             using (var myFileStream = new FileStream(fullPathName, FileMode.Open))
             {
                 theRobotVariants = (robotVariants)mySerializer.Deserialize(myFileStream);
+                myFileStream.Close();
```

Comment:
Close() is not needed. The using statement should take care of this... see https://learn.microsoft.com/en-us/dotnet/api/system.io.filestream?view=net-7.0